src/user/user.controller.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Delete,
+  Param,
+  ParseIntPipe,
+} from '@nestjs/common';
+import { CreateUserDto } from './dtos/create-user.dto';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+  constructor(private userService: UserService) {}
+  @Get('/all')
```

Comment:
```suggestion
  @Get()
```
Không đúng chuẩn Rest API. Tìm hiểu lại